oak_tensorflow_service/build.rs

```diff
@@ -14,7 +14,22 @@
 // limitations under the License.
 //
 
+use lazy_static::lazy_static;
+use std::{path::PathBuf, process::Command};
+
 const SCHEMA: &str = "schema.fbs";
+const TFLITE_BUILD_TARGET: &str = "//cc/tflite_micro:tflite-micro";
+
+lazy_static! {
```

Comment:
We don't use lazy_static any more, I think https://doc.rust-lang.org/nightly/std/cell/struct.LazyCell.html should work here?

Reply:
Removed `lazy_static`